omero/sysadmins/unix/install-web/walkthrough/omeroweb-install-centos7-ice3.6.rst

```diff
@@ -1,3 +1,6 @@
+.. walkthroughs are generated using ansible, see
```

Comment:
This will be overridden next time we run the autogen job
The changes need to be made at the level of  https://github.com/ome/omeroweb-install

Reply:
Ah, that was what I was asking before, I obviously misunderstood your answer